.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Rather than create this separate .gitignore file, please ignore the entire .idea directory.